nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -204,181 +215,40 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <properties>
```

Comment:
The top-level `properties` in this `pom.xml` are probably all redundant now - remove

Reply:
Removed